modules/lib/lib-node/src/main/resources/lib/xp/node.ts

```diff
@@ -552,7 +558,9 @@ export interface RepoConnection {
 
     setChildOrder<NodeData = Record<string, unknown>>(params: SetChildOrderParams): Node<NodeData>;
 
-    query(params: QueryNodeParams): NodeQueryResult;
+    query<
+        AggregationOutput extends Record<string, AggregationsResult> = never
```

Comment:
Shouldn't this take `AggregationInput` as a type parameter instead (which should also be a type parameter for `QueryNodeParams`), and resolve the `AggregationOutput`, like `query` in *content-lib*?

This applies to all `query` methods in this file.

Reply:
Yes, I made the two steps changes. On the second step (after adding `AggregationsToAggregationResults`) I totally forgot to update the node lib, changing the content lib only. 🙀